### PR TITLE
update id to be unique

### PR DESCRIPTION
**Pull request by JackySun9**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "test.sauce.us": "saucectl run --region us-west-1",
     "test.sauce.eu": "saucectl run",
     "postinstall": "npm update browserstack-node-sdk",
-    "test:bstack": "npx browserstack-node-sdk playwright test"
+    "test:bstack": "npx browserstack-node-sdk playwright test",
+    "test:uar": "npx playwright test --config=configs/uar.config.js --project=uar-live-chrome --headed -g"
```
Comment (Dli3) [inline package.json:19]: I don't think we want to add team specific commands here.

Comment (skumar09) [reply package.json]: I agree with @Dli3 , let's not have consuming application specific command

File: features/uar/quiz.spec.js [hunk 2]
```diff
@@ -2,74 +2,11 @@ module.exports = {
   name: 'Quiz Recommender Blocks',
   features: [
     {
-      tcid: '000',
+      tcid: '0',
       name: '@quiz basic',
       path: '/drafts/quiz/quiz-2/',
-      tags: '@cc @uar-quiz-basic @uar-quiz-static',
+      tags: '@cc @uar-quiz-basic-milo @uar-quiz-static',
```
Comment (skumar09) [inline features/uar/quiz.spec.js:8]: @JackySun9 : You may want to generalize these tags ? as they are long string [ `@uar-quiz-basic-milo`  ]

**Review by skumar09 — APPROVED** [review]
- PR Looks good

- Have few minor comments, please review them

**Pull request merged**